src/ClimaAtmos/Dycore/src/AtmosStateArrays.jl

```diff
@@ -5,6 +5,26 @@ using MPI
 
 export AtmosStateArray
 
+"""
+   AtmosStateArray{S <: Tuple, T, DeviceArray, N,
```

Comment:
The indentation should be 4 spaces so that it renders as a code block in HTML.

Reply:
thanks